lib/active_admin/middleware/error_handler.rb

```diff
@@ -0,0 +1,54 @@
+module ActiveAdmin
+  class ErrorHandler
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      @env = env
+      @app.call(env)
+    rescue Exception => exception
+      if is_active_admin_error? && use_active_admin_error_page?
+        render_exception(exception)
```

Comment:
change it to `render_exception(exception, env)`, then we don't need a `@env` var.